src/pages/CartPage/CartPage.jsx

```diff
@@ -78,7 +79,7 @@ function CartPage() {
                 </div>
                 <div className="cart-aside-container">
                     <div className="cart-aside-list">
-                        <span>Price({cartState.length} items)</span>
+                        <span>Price({cartState?.reduce((acc,cur)=>acc +=  cur.qty,0)} items)</span>
```

Comment:
Extract this logic on top like totalCartQuantity

Reply:
okay done